lib/ansiblelint/utils.py

```diff
@@ -236,6 +240,30 @@ def _taskshandlers_children(basedir, k, v, parent_type):
                 'path': path_dwim(basedir, th['include']),
                 'type': playbook_section
             })
+        elif 'include_tasks' in th:
```

Comment:
the sections for include, include_tasks and import_tasks are identical other than the path - this can be abstracted to a method or a loop